app/models/finance/schedule/npq_leadership.rb

```diff
@@ -11,6 +11,10 @@ class NPQLeadership < Schedule
         npq-executive-leadership
       ].freeze
 
+      def self.permitted_course_identifiers
+        IDENTIFIERS
+      end
```

Comment:
So we either move everything to class methods or constants, not both. For this sort of things I prefer constants.

Reply:
These are now all constants